semgrep-core/src/matching/Generic_vs_generic.ml

```diff
@@ -546,6 +552,10 @@ and m_id_info a b =
 (* experimental! *)
 and m_expr_deep a b =
   m_deep m_expr_deep m_expr SubAST_generic.subexprs_of_expr a b
+  >||> m_with_symbolic_propagation
```

Comment:
Maybe a comment on why we do it at this level. Why not in m_expr? why in m_expr_deep?

Reply:
Actually it is not needed here, just in `m_expr`, I fixed it!